### PR TITLE
Health Cap Potion

### PR DESCRIPTION
**Pull request by N8-Des**
Added the health cap potion to the wiki

### CODE REVIEW

File: wiki/HealthCapPotion [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Health Cap Potion
+=======
+* Cost:
```
Comment (jyan-zarate) [inline wiki/HealthCapPotion:3]: Use this following formatting instead. The below values are placeholders.

* Cost:
    * Construction Cost: 1
    * Buy from Pharmacy with `M` standing: 5
    * Buy from Pharmacy with `A` standing: 10
    * Buy from Pharmacy with `N` standing: 15
    * Buy from Pharmacy with `E` standing: 20

File: wiki/HealthCapPotion [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  Owner: 0
+  Ally: 50
+  Neutral: 100
+  Enemy: 250
+  
+#### Abilities
+
+The health cap potion increases the health cap (maximum health)
+of the player, it can be bought in pharmacies.
```
Comment (jyan-zarate) [inline wiki/HealthCapPotion:12]: No need for a long line, just enable word wrap in your editor.

File: wiki/HealthCapPotion [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  Enemy: 250
+  
+#### Abilities
+
+The health cap potion increases the health cap (maximum health)
+of the player, it can be bought in pharmacies. 
+
+#### Other
+
+The health cap potion can be used multiple times, but does
```
Comment (jyan-zarate) [inline wiki/HealthCapPotion:16]: Capitalize Health Cap Potion.

File: wiki/HealthCapPotion [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  
+#### Abilities
+
+The health cap potion increases the health cap (maximum health)
+of the player, it can be bought in pharmacies. 
+
+#### Other
+
+The health cap potion can be used multiple times, but does 
+not increase the cap by the same amount each time it is used.
```
Comment (jyan-zarate) [inline wiki/HealthCapPotion:17]: No need for a long line, just enable word wrap in your editor.

**Review by JCharante — CHANGES_REQUESTED** [review]
Please address these concerns, and then I'll review the `pr` again.

File: wiki/HealthCapPotion [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+of the player, it can be bought in pharmacies. 
+
+#### Other
+
+The health cap potion can be used multiple times, but does 
+not increase the cap by the same amount each time it is used. 
+
+Contributors
+============
+-Nathan Desjardins
```
Comment (jyan-zarate) [inline wiki/HealthCapPotion:21]: Please add a line break between line 20 & 21.

**Pull request closed without merging**